changelog/_unreleased/2021-07-29-entity-before-write-event.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Added new event to validate entity payload before write
+issue: 1983
+author: Philipp Georg
+author_email: info@moorleiche.com
+author_github: moorl
+---
+# Core
+* new class `\Shopware\Core\Framework\DataAbstractionLayer\Event\EntityWritePayloadEvent`
```

Comment:
Please use the corresponding keywords as ducomented [here](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-Implement-New-Changelog.md).